Content.Server/Administration/Systems/AdminVerbSystem.Tools.cs

```diff
@@ -65,343 +65,340 @@ private void AddTricksVerbs(GetVerbsEvent<Verb> args)
         if (!_adminManager.HasAdminFlag(player, AdminFlags.Admin))
             return;
 
-        if (_adminManager.HasAdminFlag(player, AdminFlags.Admin))
+        if (TryComp<DoorBoltComponent>(args.Target, out var bolts))
         {
-            if (TryComp<DoorBoltComponent>(args.Target, out var bolts))
+            Verb bolt = new()
             {
-                Verb bolt = new()
+                Text = bolts.BoltsDown ? "Unbolt" : "Bolt",
+                Category = VerbCategory.Tricks,
+                Icon = bolts.BoltsDown
+                    ? new SpriteSpecifier.Texture(new("/Textures/Interface/AdminActions/unbolt.png"))
+                    : new SpriteSpecifier.Texture(new("/Textures/Interface/AdminActions/bolt.png")),
+                Act = () =>
                 {
-                    Text = bolts.BoltsDown ? "Unbolt" : "Bolt",
-                    Category = VerbCategory.Tricks,
-                    Icon = bolts.BoltsDown
-                        ? new SpriteSpecifier.Texture(new("/Textures/Interface/AdminActions/unbolt.png"))
-                        : new SpriteSpecifier.Texture(new("/Textures/Interface/AdminActions/bolt.png")),
-                    Act = () =>
-                    {
-                        _door.SetBoltsDown((args.Target, bolts), !bolts.BoltsDown);
-                    },
-                    Impact = LogImpact.Medium,
-                    Message = Loc.GetString(bolts.BoltsDown
-                        ? "admin-trick-unbolt-description"
-                        : "admin-trick-bolt-description"),
-                    Priority = (int) (bolts.BoltsDown ? TricksVerbPriorities.Unbolt : TricksVerbPriorities.Bolt),
-                };
-                args.Verbs.Add(bolt);
-            }
+                    _door.SetBoltsDown((args.Target, bolts), !bolts.BoltsDown);
+                },
+                Impact = LogImpact.Medium,
+                Message = Loc.GetString(bolts.BoltsDown
+                    ? "admin-trick-unbolt-description"
+                    : "admin-trick-bolt-description"),
+                Priority = (int) (bolts.BoltsDown ? TricksVerbPriorities.Unbolt : TricksVerbPriorities.Bolt),
```

Comment:
```suggestion
                Priority = (int)(bolts.BoltsDown ? TricksVerbPriorities.Unbolt : TricksVerbPriorities.Bolt),
```
Since we are messing up the git history anyways we might as well fix the formatting.
Same below